sormas-ui/src/main/java/de/symeda/sormas/ui/utils/UserField.java

```diff
@@ -50,6 +53,7 @@ protected Component initContent() {
 		}
 		userCombo.setNullSelectionAllowed(true);
 		userCombo.setValue(getValue());
+		parentPseudonymizedFlag = parentPseudonymizedSupplier.get();
```

Comment:
getting the parent pseudonymization flag here is useless here

`parentPseudonymizedSupplier.get()` should be called only when needed, when sowing the details popup

Reply:
removed and moved in the pop-up window code